docs/development/commands.md

```diff
@@ -0,0 +1,137 @@
+# Creating your first command
+
+This tutorial focuses on client commands, meaning they will get run on the client. If you want to develop a server command there are more considerations to be done (like permissions, and synchronizing server state to the client).
+
+## Basic command class
+
+First, let's create a new class for our command. We will call it `CrashCommand` because it will crash your game. Of course, your command can do whatever you want. We need to make sure our command `extends CommandBase`.
+
+```java
+public class CrashCommand extends CommandBase {
+
+    @Override
+    public String getCommandName() {
+        return "crashme";
+    }
+
+    @Override
+    public String getCommandUsage(ICommandSender sender) {
+        return "";
+    }
+
+    @Override
+    public void processCommand(ICommandSender sender, String[] args) throws CommandException {
+        throw new RuntimeException("Not yet implemented!");
+    }
+
+    @Override
+    public boolean canCommandSenderUseCommand(ICommandSender sender) {
+        return true;
+    }
+
+}
+```
+
+I already implemented three methods into the command.
+
+!!! warning
+    When writing a client command you will need to override `canCommandSenderUseCommand`. By default this method does not generate, but without it you will get a `You do not have permission to use this command` error (since you by default do not have any permissions on a server). Just always return `true`, since the command is client side only anyway.
+
+
+First the method `getCommandName`, which returns the name of your command. The name is what you use in chat to run the command. The command name should be just numbers and letters (and maybe dashes, if you want). In chat you will need to run `/crashme` to run this command (the `/` gets added to the name automatically).
+
+The second method is `getCommandUsage`. You need to override it because every commands needs to have a usage, but for most SkyBlock mods that usage doesn't matter — it only get's displayed in the `/help` menu, and Hypixel does not show client commands in it's help menu.
```

Comment:
`get's` -> `gets`
The correct usage in your sentence would be "gets" without an apostrophe. The word "gets" is the third person singular form of the verb "get" and does not require an apostrophe in this context.